### ISSUE TITLE
Possible improvement: use marcxml or mods schema instead of dc

### ISSUE THREAD

**Issue by arangb** [xlcnd/isbnlib-loc#1]
Hello,
I have not done any extensive testing, but with the three ISBNs provided in the test_metadata.py file, only one returns a complete result:

9780321534965 --> ok! all fields are found
9781118241257 --> no publisher nor year (ok if we use goob)
9780425284629 --> no publisher nor year (no publisher if we use goob)

Indeed, I checked that in _loc.py the SERVICE_URL asks for `&recordSchema=dc` and the info returned does NOT contain the publisher/date fields for those two ISBNs!
[9781118241257 with dc](http://lx2.loc.gov:210/lcdb?version=2.0&operation=searchRetrieve&query=bath.isbn=9781118241257&maximumRecords=1&recordSchema=dc)
[9780425284629 with dc](http://lx2.loc.gov:210/lcdb?version=2.0&operation=searchRetrieve&query=bath.isbn=9780425284629&maximumRecords=1&recordSchema=dc)
It's not that the parser is failing: the record is incomplete!

However, if you ask for `&recordSchema=marcxml`, both entries contain the publisher and year.
[9781118241257 with marcxml](http://lx2.loc.gov:210/lcdb?version=2.0&operation=searchRetrieve&query=bath.isbn=9781118241257&maximumRecords=1&recordSchema=marcxml)
[9780425284629 with marcxml](http://lx2.loc.gov:210/lcdb?version=2.0&operation=searchRetrieve&query=bath.isbn=9780425284629&maximumRecords=1&recordSchema=marcxml)

Again, I don't know how often this is a problem, or if there any deeper reasons to use the dc schema, but perhaps it deserves another look to consider changing the schema and parser.
There is a [pymarc](https://pypi.org/project/pymarc/) project in PyPi  that maybe could be used to parse the marcxml result. 
 The LoC also offers the newer MODS schema that contains all the info correctly in the examples above. So that could be used if it is preferred over MARCXML.

**Comment by xlcnd** [comment]
Hi,

Thanks again.

When I evaluate which schema to use, I ruled out the _marcxml_ because:

1.  I saw that a lot of times the call to the service timeout and got many instances of 'service not available' (probably they have many applications using this service and is slower to respond?!)
2. The parsing gets complicated, because the MARC format _and_ the many inconsistencies in the marking (sometimes the same information is in one field and other times is in another...)

In the end I opted for the _dc_ service.

About the MODS... I wasn't aware of that, let me investigate that!
